Engine/src/jaxb/unmarshal/converter/expression/converter/ExpressionConverterAndValidator.java

```diff
@@ -227,12 +227,9 @@ private boolean compareIntegerOrDoubleCase(PRDAction prdAction){
         Entity entity = entities.get(entityName);
         ActionType type = ActionType.valueOf(actionType);
         PropertyType propertyType;
-        boolean ret = true;
+        boolean ret = type == ActionType.INCREASE || type == ActionType.DECREASE || type == ActionType.CALCULATION || type == ActionType.CONDITION;
```

Comment:
This should be an if statement in order to catch this specific error and add it to the error list. Return the condition value wont help us.